bolt/test/runtime/X86/hot-end-symbol.s

```diff
@@ -30,12 +31,12 @@
 # CHECK-OUTPUT:       __hot_start
 # CHECK-OUTPUT-NEXT:  main
 # CHECK-OUTPUT-NEXT:  __hot_end
+# CHECK-OUTPUT-NOT:   __hot_start.cold
 
   .text
   .globl  main
   .type main, %function
   .globl  __hot_start
-  .type __hot_start, %object
```

Comment:
why are you removing this?

Reply:
If __hot_start has `%object` type, it will be an absolute symbol and won't belong to .text section. When updating function symbols, we check that symbol section matches that of the function. So we need hot_start to be a global symbol in `.text` section.